packages/storybook/src/css/Page.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Meta, StoryObj } from '@storybook/react';
+import '@gemeente-denhaag/page';
+
+const meta = {
+  title: 'CSS/Page Elements/Page',
```

Comment:
Hier mist misschien nog wat params?

Reply:
Volgens mij niet, de oorspronkelijke story had ook al weinig informatie: https://raw.githubusercontent.com/nl-design-system/denhaag/1e2e93aa678ecb31c1be6b4b069ea36b5509f703/packages/storybook/src/css/Page.stories.mdx. Misschien willen we dit in de toekomst aanvullen met wat documentatie?